samples/nrf9160/gps/src/main.c

```diff
@@ -35,7 +35,17 @@
 #elif CONFIG_GPS_SAMPLE_ANTENNA_EXTERNAL
 #define AT_COEX0       "AT\%XCOEX0"
 #endif
+#endif /* CONFIG_BOARD_NRF9160DK_NRF9160NS */
+
+#ifdef CONFIG_BOARD_THINGY91_NRF9160NS
+#define AT_MAGPIO      "AT\%XMAGPIO=1,1,1,7,1,746,803,2,698,748,2,1710,2200," \
+			"3,824,894,4,880,960,5,791,849,7,1565,1586"
+#ifdef CONFIG_GPS_SAMPLE_ANTENNA_ONBOARD
+#define AT_COEX0       "AT\%XCOEX0=1,1,1570,1580"
```

Comment:
I think you need to update also [line 53](https://github.com/nrfconnect/sdk-nrf/pull/3077/files#diff-58f3c427fa4c71d51808275ce17d3f89R53) so that those macros are actually used.

Reply:
Updated, that was a bad miss..